resize.php

```diff
@@ -0,0 +1,71 @@
+<?php
+require(dirname(__FILE__).'/config/config.inc.php');
```

Comment:
Unfortunately, we can't accept a script like that, it has to be executed by a Symfony Controller

Reply:
@PierreRambaud
> Unfortunately, we can't accept a script like that, it has to be executed by a Symfony Controller
As far as I understand, Symfony Controllers are used in the Admin Area. This script must run in the frontend. I think you are not speaking of a FrontController, do you? Or do you speak of a ModuleFrontController?
Could you elaborate or hint me to some explanatory doc?

Reply:
Indeed, if this script must run in the FO, we can't use Symfony.

Symfony router is called there https://github.com/PrestaShop/PrestaShop/blob/develop/admin-dev/index.php which is the admin area.

Reply:
That being said, it would be better to put all the logic in a Front controller in `front/Controller` I think 🤔